ci/checks/style.sh

```diff
@@ -66,4 +66,21 @@ RETVALS=($ISORT_RETVAL $BLACK_RETVAL $FLAKE_RETVAL $FLAKE_CYTHON_RETVAL)
 IFS=$'\n'
 RETVAL=`echo "${RETVALS[*]}" | sort -nr | head -n1`
 
+# Check for a consistent code format
+# TODO: keep adding more dirs when we add more source folders in cuml
+FORMAT=`python cpp/scripts/run-clang-format.py 2>&1`
+FORMAT_RETVAL=$?
+if [ "$RETVAL" = "0" ]; then
+  RETVAL=$FORMAT_RETVAL
+fi
+
+# Output results if failure otherwise show pass
+if [ "$FORMAT_RETVAL" != "0" ]; then
+  echo -e "\n\n>>>> FAILED: clang format check; begin output\n\n"
+  echo -e "$FORMAT"
+  echo -e "\n\n>>>> FAILED: clang format check; end output\n\n"
+else
+  echo -e "\n\n>>>> PASSED: clang format check\n\n"
+fi
```

Comment:
These need to go about the `RETVALS` handling above in lines 65 - 67. Also suggest renaming `$FORMAT_RETVAL` to `$CLANG_RETVAL` or something similar to be more explicit.

Reply:
Done!